### PR TITLE
Added Ansible UTM Module for managing network aaa entries.

### PR DESCRIPTION
**Pull request by maunzCache**
##### SUMMARY
This PR will add a new utm module which allows the management of network aaa entries of the Sophos UTM. This module integrates with the existing utm_utils .

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
- module/web_infrastructure/sophos_utm/utm_network_aaa.py

### CODE REVIEW

**Comment by MatrixCrawler** [comment]
bot_status

**Review by stearz — APPROVED** [review]

**Comment by stearz** [comment]
shipit

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_network_aaa.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+---
+module: utm_network_aaa
+
+author:
+    - Stephan Schwarz (@stearz)
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_network_aaa.py:22]: Please update this to your name

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_network_aaa.py (hunk 1 above)
Comment (stearz) [reply lib/ansible/modules/web_infrastructure/sophos_utm/utm_network_aaa.py:22]: Sorry? This is my name. What am I missing?

Comment (maunzCache) [reply lib/ansible/modules/web_infrastructure/sophos_utm/utm_network_aaa.py:22]: Same as the other PR. @stearz did the code but i commited it in his name so he is the author. We're working colleagues and i was reviewing the code.

**Comment by getjack** [comment]
I am very sorry but i deleted the remote repo and i didn't check the PRs beforehand. Will redo the PRs and fix it. Sorry for the overhead :(

**Pull request closed without merging**